tests/test_multidict.py

```diff
@@ -296,104 +300,105 @@ def test__repr__(self):
         d = self.make_dict()
         cls = self.proxy_cls if self.proxy_cls is not None else self.cls
 
-        self.assertEqual(str(d), "<%s()>" % cls.__name__)
+        assert str(d) == "<%s()>" % cls.__name__
         d = self.make_dict([('key', 'one'), ('key', 'two')])
-        self.assertEqual(
-            str(d),
+        assert(
+            str(d) ==
             "<%s('key': 'one', 'key': 'two')>" % cls.__name__)
```

Comment:
You'd better do `expected = "<%s('key': 'one', 'key': 'two')>" % cls.__name__` one line above instead of having multiline expression for the sake of readability